tests/resources/sample_pandas_datetime_schema.json

```diff
@@ -0,0 +1,27 @@
+{
+    "type": "object",
+	"properties": {
+		"param": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": [
+                    "datetime"
+                ],
+                "properties": {
+                    "datetime": {
+                        "type": "string",
+                        "format": "date-time"
+                    }
+                }
+            }
+        }
+    },
+    "example": {
+        "param": [
+            {
+                "datetime": "2013-12-31T00:00:00.000Z"
```

Comment:
How about including timedelta in this test, since you added it in the Swagger generator code?

Reply:
Good call, turned out there was a whole other issue with timedelta stuff, that should be fixed now.